README.md

```diff
@@ -0,0 +1,24 @@
+# Backend-registry
+Backend for Register of Clinical Practice projects of St. Petersburg State University.
+
+## Launch
+1. Download [Docker](https://docs.docker.com/engine/install/)
+2. Open src -> main -> docker and run:
+  ```java
+  docker-compose up
+  ```
+  
+## Dev notes
+1. Download [Java17](https://www.oracle.com/cis/java/technologies/downloads/) and [Maven](https://maven.apache.org/download.cgi)
+2. In resources -> application properties you can find credentials to deploy the database. 
+
+
+## Database 
+Information and description about database is located and in *DataBase* folder.
+
+## Software testing
+
+[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/20640621-98eced96-51b4-42e0-9eab-d6b96913f4dd?action=collection%2Ffork&source=rip_markdown&collection-url=entityId%3D20640621-98eced96-51b4-42e0-9eab-d6b96913f4dd%26entityType%3Dcollection%26workspaceId%3D2b29e6e1-84ae-4a9a-889b-8218192e6351)
+
+## Asana
+We used Asana for task management throughout the duration of the project. All information is shown in **Project managment** folder.
```

Comment:
Change to `\project_management\`. Please rename the corresponding folder in your 2nd branch